### PR TITLE
[14.0] [MIG] partner_multi_company

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration
@ForgeFlow

Depends on:

- https://github.com/OCA/multi-company/pull/270

### CODE REVIEW

**Comment by francesco-ooops** [comment]
@JasminSForgeFlow hi, can you check this since base_multi_company was merged?

**Comment by JasminSForgeFlow** [comment]
> @JasminSForgeFlow hi, can you check this since base_multi_company was merged?

Hi @francesco-ooops,

I have rebased it.

**Comment by francesco-ooops** [comment]
> I have rebased it.

Thanks! 

Do you plan on fixing travis any time soon? just to know

**Comment by JasminSForgeFlow** [comment]
> > I have rebased it.
> 
> Thanks!
> 
> Do you plan on fixing travis any time soon? just to know

Travis issue is from another module, have to check it.

**Comment by elvise** [comment]
Hi @JasminSForgeFlow any plan for this task ?

**Comment by elvise** [comment]
> > > I have rebased it.
> > 
> > 
> > Thanks!
> > Do you plan on fixing travis any time soon? just to know
> 
> Travis issue is from another module, have to check it.

Hi @JasminSForgeFlow just a gentle reminder :)

**Review by elvise — APPROVED** [review]
Functional Review: OK!

**Comment by elvise** [comment]
@pedrobaeza also this one from forgeflow looks good :)

**Comment by pedrobaeza** [comment]
Sorry, not following this module. Needs more approvals.

/ocabot migration partner_multi_company

**Comment by pedrobaeza** [comment]
And CI is not green.

**Comment by elvise** [comment]
> And CI is not green.

Yup, I’m sorry!

**Comment by elvise** [comment]
@JasminSForgeFlow can you check CI ?

**Comment by francesco-ooops** [comment]
@JasminSForgeFlow do you plan on working on this anytime soon? just to know :)

**Comment by francesco-ooops** [comment]
@JasminSForgeFlow another try with tests? :)

**Comment by GSLabIt** [comment]
@JasminSForgeFlow any chance to rebase?

**Comment by JasminSForgeFlow** [comment]
> @JasminSForgeFlow any chance to rebase?

Rebase won't work, need to debug issue.

**Comment by francesco-ooops** [comment]
@JasminSForgeFlow any luck with debugging?

**Comment by JasminSForgeFlow** [comment]
> @JasminSForgeFlow any luck with debugging?

Nope :(

**Comment by francesco-ooops** [comment]
> > @JasminSForgeFlow any luck with debugging?
> 
> Nope :(

can you share your findings and maybe we can help?

**Comment by JasminSForgeFlow** [comment]
> > > @JasminSForgeFlow any luck with debugging?
> > 
> > 
> > Nope :(
> 
> can you share your findings and maybe we can help?

When we try to run tests with purchase_sale_inter_company, one of the test cases of purchase_sale_inter_company is failing, because it has test something like PO is created with one user and invoice is made with other user and while posting invoice getting an issue.

It could be related to some updates happening in this module while the creation of user and partners

**Comment by francesco-ooops** [comment]
@rousseldenis any hint on this?

**Comment by francesco-ooops** [comment]
@OCA/intercompany-maintainers any suggestion?

**Comment by francesco-ooops** [comment]
@JordiBForgeFlow would you have an insight on this one? :)

File: partner_multi_company/tests/test_partner_multi_company.py
Comment (ilyasProgrammer) [inline partner_multi_company/tests/test_partner_multi_company.py:149]: When you change user's company it should not change related partner's company. I think here should be just self.user_company_1.company_id

**Review by ilyasProgrammer — CHANGES_REQUESTED** [review]

File: partner_multi_company/tests/test_partner_multi_company.py
Comment (francesco-ooops) [reply partner_multi_company/tests/test_partner_multi_company.py:149]: @JasminSForgeFlow can you check this? :)

**Comment by pedrobaeza** [comment]
Superseded by #411

**Pull request closed without merging**
